k8s/yaml/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ts-bridge
+spec:
+  selector:
+    matchLabels:
+      app: ts-bridge
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: ts-bridge
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+      containers:
+        - image: REPLACE_ME
+          imagePullPolicy: Always
+          name: ts-bridge
+          env:
+            # Use secret in real usage
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/gcp/google_key.json
+          args:
+            - --storage-engine=boltdb
+            - --enable-status-page
+            - --stats-sd-project=REPLACE_ME
+            - --update-parallelism=4
+            - --sync-period=10s
+          ports:
+            - containerPort: 8080
+              name: ts-bridge
+          resources:
+            limits:
+              memory: "1Gi"
+            requests:
+              memory: "500Mi"
+          livenessProbe:
```

Comment:
A readinessProbe will be good as well so the rollout can be checked. Also I believe readinessProbe is required for GCLB to balance to it if that's ever required.